EIPS/eip-3076.md

```diff
@@ -0,0 +1,292 @@
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-27
+---
+
+## Simple Summary
+
+A JSON interchange format for Serenity (eth2) that contains the necessary slashing protection information required to safely migrate keys between clients.
+
+## Abstract
+
+ A standard format for transferring a key's signing history allows validators to easily switch between clients without the risk of signing conflicting messages. While a [common keystore format](https://eips.ethereum.org/EIPS/eip-2335) provides part of the solution, it does not contain any information about a key's signing history. For a validator moving their keys from client A to client B, this can lead to scenarios in which client B inadvertently signs a message that conflicts with an earlier message signed with client A.  
+ 
+ We propose a database interchange format that helps solve this problem. The format contains a record of all blocks and attestations signed by a set of validators. 
+
+
+
+## Motivation
+
+Eth2 penalises validators for voting in ways that could result in two different versions of the chain being finalised. These types of penalties are called slashings.
+
+> The two main ways a validator can be slashed are 
+>
+> 1. **Double voting**: voting for two different blocks during the same epoch.
+> 2. **Surround voting**: voting for one version of reality, and later voting for another version in a way that doesn't make it clear that they no longer believe in the first.
+
+For a validator following the protocol correctly, there is, in principle, no risk of being slashed. However, changing clients (from client A to client B, say) can result in a slashing risk if client B is unaware of the blocks and attestations that were signed with client A. 
+
+Specifically, problems can occur if client A and client B do not agree on what the present time is (i.e if their clocks are not synchronised within a certain bound).
+
+For example, say client A's time is accidentally set to a day in the future (225 epochs), and a validator switches from client A to client B without giving B a record of the blocks and attestations signed with A.
+
+The validator in question now runs the risk of voting for two different blocks in the same epoch (a slashable offence) for the next 225 epochs (since they've already voted on these epochs with client A, and now stand to vote on them again with client B).
+
+The above situation may sound far-fetched, but it's actually very similar to the incident that occurred on the eth2 Medalla testnet on August 14th, where a time skew that affected all Prysm clients spiraled into a series of [cascading failures](https://medium.com/prysmatic-labs/eth2-medalla-testnet-incident-f7fbc3cc934a). It resulted in the testnet experiencing several days without finality -- an outcome that could have resulted in tens of millions of dollars in collective penalties had it occurred on mainnet. 
```

Comment:
```suggestion
The above situation may sound far-fetched, but it's actually very similar to the incident that occurred on the eth2 Medalla testnet on 2020-08-14, where a time skew that affected all Prysm clients spiraled into a series of [cascading failures](https://medium.com/prysmatic-labs/eth2-medalla-testnet-incident-f7fbc3cc934a). It resulted in the testnet experiencing several days without finality -- an outcome that could have resulted in tens of millions of dollars in collective penalties had it occurred on mainnet.
```

Dates should include years, since we don't know what year a reader will be reading this in.  Also, I have a personal preference for ISO-8601 formatted dates which is why this suggestion switches it, but feel free to stick with some other format if you prefer.